gather-scheduler.js

```diff
@@ -28,6 +28,8 @@ class GatherScheduler {
   static run(gatherers, options) {
     const driver = options.driver;
     const url = options.url;
+    const loadPage = options.flags.loadPage;
```

Comment:
It's optional, the CLI defaults to `true`, the extension doesn't, unless you tell it to reload the page.